### ISSUE TITLE
Move b3 under propagation/b3?

### ISSUE THREAD

**Issue by rakyll** [open-telemetry/opentelemetry-go#1128]
Currently, the B3 propagation implementation is in the trace API. Would it better to move it under propagation/b3 as an implementation of the propagation interfaces? Not many users unless required will depend on B3 and it confuses uses as it's the default propagation format.

**Comment by MrAlias** [comment]
Duplicate of https://github.com/open-telemetry/opentelemetry-go/issues/1012 and being addressed in https://github.com/open-telemetry/opentelemetry-go/pull/1118

Please reopen if I've misunderstood.

**Comment by rakyll** [comment]
It is a duplicate, sorry for that :)
